model_compression_toolkit/quantizers_infrastructure/README.md

```diff
@@ -0,0 +1,93 @@
+# Quantizers Infrastructure (QI)
+
+Quantizers infrastructure is a library that provides quantization modules for hardware-oriented model optimization tools.
+
+The quantization modules (quantizers) can be used for emulating inference-time quantization (inferable-quantizers) and for optimization of model quantization during training (trainable quantizers).
+
+## Library structure
+
+### Quantizer
+
+Quantizer instance implements the quantization function: conversion of floating point values to quantized values represented in floating point (fake-quantization).
```

Comment:
1. A\The
2. The quantizer quantizes an input tensor that might be float (weights) or quantized (most activations)